Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -35,30 +37,42 @@ class BrowserViewController: UIViewController {
         tabManager = TabManager(defaultNewTabRequest: defaultRequest)
     }
 
-    override func viewDidLoad() {
-        let headerView = UIVisualEffectView(effect: UIBlurEffect(style: UIBlurEffectStyle.ExtraLight))
-        view.addSubview(headerView);
-
-        headerView.snp_makeConstraints { make in
-            make.top.equalTo(self.view.snp_top)
-            make.height.equalTo(ToolbarHeight + StatusBarHeight)
+    private func wrapInEffect(view: UIView, position: String) {
+        let effect = UIVisualEffectView(effect: UIBlurEffect(style: UIBlurEffectStyle.ExtraLight))
+        self.view.addSubview(effect);
+
+        effect.snp_makeConstraints { make in
+            if position == "top" {
```

Comment:
The function taking a string to do layout like this is kind of awkward. Suggestion:
* Remove the <code>position</code> argument
* Return the effect
* Set the constraints on the effect in <code>viewDidLoad</code>